vortexasdk/endpoints/voyages_search_enriched.py

```diff
@@ -293,4 +296,7 @@ def search(
             "vessel_risk_level_excluded": convert_to_list(vessel_risk_level_excluded),
         }
 
-        return VoyagesSearchEnrichedResult(super().search(headers=self._HEADERS, **api_params))
+        if columns is None:
+            return VoyagesSearchEnrichedListResult(super().search(**api_params))
+        else:
+            return VoyagesSearchEnrichedFlattenedResult(super().search(headers=self._HEADERS, **api_params))
```

Comment:
I would rename `HEADERS` to `CSV_HEADERS` so there is no doubt about why we pass them only in one case.